packages/swc/src/type.ts

```diff
@@ -0,0 +1,3 @@
+import type { Options as SWCOptions } from '@swc/core';
+
+export type Options = SWCOptions & {};
```

Comment:
I’d suggest instead of making the top level the swc options object making the root object have a `swc` property, and have that be the thing that is passed to `swc`

This means if plug-in specific options get added in the future they are separate to the swc options and there can’t be any collisions

Reply:
good point :ok_hand: